frontend/src/i18n/locales/de.yaml

```diff
@@ -747,6 +747,8 @@ api-remote-errors:
   series:
     not-found: "Aktion fehlgeschlagen: Serie nicht gefunden."
     not-allowed: Sie haben nicht die Berechtigung, diese Serienaktion durchzuführen.
+    placeholder:
+      not-found: "Nicht gefunden. Die Platzhaltererstellung ist vermutlich fehlgeschlagen."
```

Comment:
And in that case, this error message can also be removed I think (it was in the wrong spot anyway?)